gstreamer-player/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "gstreamer-player"
+version = "0.1.0"
+authors = ["Sebastian Dröge <sebastian@centricular.com>"]
+categories = ["api-bindings", "multimedia"]
+description = "Rust bindings for GStreamer Player library"
+repository = "https://github.com/sdroege/gstreamer-rs"
+license = "MIT/Apache-2.0"
+documentation = "https://gstreamer.freedesktop.org"
+keywords = ["gstreamer", "multimedia", "audio", "video", "gnome"]
+build = "build.rs"
+
+[dependencies]
+bitflags = "0.9"
+libc = "0.2"
+glib-sys = { version = "0.3.4", git = "https://github.com/gtk-rs/sys" }
+gobject-sys = { version = "0.3.4", git = "https://github.com/gtk-rs/sys" }
+gstreamer-sys = { version = "0.1.1", git = "https://github.com/sdroege/gstreamer-sys", features = ["v1_8"] }
+gstreamer-player-sys = { version = "0.1.1", git = "https://github.com/sdroege/gstreamer-sys", features = ["v1_12"] }
+glib = { version = "0.1.3", git = "https://github.com/gtk-rs/glib" }
+gstreamer = { version = "0.1.0", path = "../gstreamer" }
```

Comment:
And here for gstreamer-sys and gstreamer, requiring v1_12 would be good for consistency